### DISCUSSION TITLE
No 2.5.2 file available for ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by stavekoff** [abraunegg/onedrive#2947] [Q&A]
am i missing something? im new to linux and am stupid

**Comment by abraunegg** [comment]
@stavekoff 
Please read all documentation to correctly install the client on your platform, or install a better Linux distribution such as Fedora that keeps packages up-to-date.

**Comment by AtticusZeller** [comment]
https://github.com/abraunegg/onedrive/blob/master/docs%2Fubuntu-package-install.md

**Reply by stavekoff** [reply]
THANK YOU FOR HELP

**Comment by abraunegg** [comment]
@andrea 
> I'm using Fedora and DNF does not pick up latest 2.5.2 version (last found is 2.4.25-3.fc39) , what am I missing? Everything else is up to date and cannot run older versions as (understandably) bugs start to arise.

As per https://repology.org/project/onedrive/versions - Fedora 39 only has v2.4.25 , as as per [Koji](https://koji.fedoraproject.org/koji/packageinfo?packageID=26044) the only platforms to be kept up-to-date are:
* FC40
* FC41
* FC42
* EL8
* EL9

Fedora 39 is currently scheduled to reach end-of-life (EOL) on 12th November 2024 .. (next week) so my advice is that you upgrade your soon to be EOL platform to Fedora 40 or above.

**Comment by steveklett** [comment]
@abraunegg Your installation guides and documentation are superb and very appreciated. I used to have that job long ago and I know how time consuming it can be.
I'm a Linux newbie (a couple days) and have read all the guides and instructions I can find to get your OneDrive client running on my Mint 22 installation. Things were going fine until I was attempting to add my first sharepoint shared library.

From https://github.com/abraunegg/onedrive/blob/master/docs/sharepoint-libraries.md:
_Before reading this document, please ensure you are running application version 2.5.3 or greater. Use onedrive --version to determine what application version you are using and upgrade your client if required._

I'm uncertain if I can update to 2.5.3 or if that is impossible due to my distro not having current packages. I barely know what I'm even saying here, I apologize. Am I out of luck or is there a way to update now that I followed your detailed installation instructions? I -think- I can't update because where would it update from? The Mint repo is out of date (as you've indicated in other comments elsewhere)

**Reply by abraunegg** [reply]
@steveklett 
> Mint 22 installation

Please install the client the correct way for your platform. When you do this, you get the correct version as per the documentation.

Do not use the distribution provided packages.

**Reply by steveklett** [reply]
Hi @abraunegg I did follow the instructions which for Mint 22 resolved to [Ubuntu 24.04](https://github.com/abraunegg/onedrive/blob/master/docs%2Fubuntu-package-install.md#distribution-ubuntu-2404)
After I completed that process ( I did every step, even the bash script to clean out old stuff and reboot) I have onedrive version `onedrive v2.4.25-1build5` installed and this is what I'm concerned about; Is this the expected outcome of a successful the installation process or did I make a mistake somewhere?

**Reply by abraunegg** [reply]
You have 100% made a mistake somewhere

**Reply by steveklett** [reply]
Indeed I did make a mistake somewhere.  I went through the whole process again and successfully installed version `v2.5.3-1+np1+1.1`
